.github/workflows/release.yml

```diff
@@ -5,6 +5,9 @@ on:
     branches:
       - main
 
+env:
+  COREPACK_DEFAULT_TO_LATEST: 0
```

Comment:
I believe this stops existing, pinned builds from bricking if `latest` has an unrecognised key.

We'll probably need to update Corepack regardless as a next step to avoid being stuck on `pnpm@9.15.3`.